buildkite/pipeline.yml

```diff
@@ -0,0 +1,19 @@
+steps:
+    - label: ":construction_worker: Make migrate image"
+      command: buildkite/make-migrate-image.sh
+
+    - wait
+
+    - label: ":mag: Build and test app"
+      command: buildkite/build-app.sh
+
+    - wait
+
+    - label: ":mag: Run smoke test"
```

Comment:
Not sure if this is the right level of parallel steps? e.g. could do smoke test before custom config tests.